src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
Classes should not start with an empty line. Please remove the empty line at the beginning of the class definition.